### PR TITLE
Update clojure_action.py

### PR DESCRIPTION
**Pull request by jelinwils**
Added the logic to check if the string for license is in the result and added it to another list called license_vulns.

### CODE REVIEW

File: clojure_action.py [hunk 1]
```diff
@@ -265,14 +265,20 @@ def _setOutput(name:str, value:str):
...
         logging.error(f"Error parsing licenses!")
     try:
         for vuln in test_res['vulnerabilities']:
-            vulns.add(VulnReport(vuln))
+            report=str(VulnReport(vuln))
+            licenseString='snyk:lic'
+            if(licenseString in report):
+                license_vulns.add(report)
+            else:
+                vulns.add(report)
```
Comment (LivingInSyn) [inline clojure_action.py:281]: You're changing the behavior here. Vulns is being changed from a `VulnReport` to a `String`. We can accomplish the same thing with a check here like:

```
if vuln['id'].startswith('snyk:lic:')
```

**Review by LivingInSyn — CHANGES_REQUESTED** [review]

File: clojure_action.py (hunk 1 above)
Comment (jelinwils) [reply clojure_action.py]: Changed the commented lines to: 
```
if(vulns['id'].startswith('snyk:lic:') is False):
```
If the license vulnerabilities need to be anywhere in the future I can add the license_vulns back but right now it did not have a purpose.

File: clojure_action.py [hunk 2]
```diff
@@ -272,12 +268,12 @@ def _setOutput(name:str, value:str):
         logging.error(f"Error parsing licenses!")
     try:
         for vuln in test_res['vulnerabilities']:
-            vulns.add(VulnReport(vuln))
+            if(vulns['id'].startswith('snyk:lic:') is False):
```
Comment (LivingInSyn) [inline clojure_action.py:271]: gonna be nitpicky one more time, sorry:

```python
if not vulns['id'].startswith('snyk:lic:'):
```

Comment (jelinwils) [reply clojure_action.py:271]: Its good to be nitpicky with this stuff! Thanks for the feedback! I will reflect the change.

Comment (jelinwils) [reply clojure_action.py]: Changed to: 
`if not vuln['id'].startswith('snyk:lic:'):`
Back to the correct variable.

**Review by LivingInSyn — APPROVED** [review]

**Pull request merged**